### PR TITLE
Change the default for "auto_play_gif" to false

### PR DESCRIPTION
**Pull request by packetcat**
This is per issue #3876 , I've mentioned my reasoning there.

https://github.com/tootsuite/mastodon/issues/3876#issuecomment-313975899

### CODE REVIEW

**Review by poopsuite — APPROVED** [review]
LGTM

File: docs/Running-Mastodon/Production-guide.md [hunk 1]
```diff
@@ -1 +0,0 @@
-[The documentation has moved to its own repository](https://github.com/tootsuite/documentation/blob/master/Running-Mastodon/Production-guide.md)
```
Comment (Gargron) [inline docs/Running-Mastodon/Production-guide.md]: Unrelated?

**Review by Gargron — CHANGES_REQUESTED** [review]

File: docs/Running-Mastodon/Production-guide.md (hunk 1 above)
Comment (staticsafe) [inline docs/Running-Mastodon/Production-guide.md]: hm looks like I fucked up syncing my fork to master, is there something I can do to fix it?

Comment (staticsafe) [inline docs/Running-Mastodon/Production-guide.md]: @Gargron added the file back with new commit

Comment (Gargron) [inline docs/Running-Mastodon/Production-guide.md]: You could just use the GitHub web editor to make this one line change.

Comment (staticsafe) [inline docs/Running-Mastodon/Production-guide.md]: Yep, done, @Gargron

Comment (Gargron) [inline docs/Running-Mastodon/Production-guide.md]: I mean this PR kinda of has a lot of unrelated commits from your fork and the unrelated file change still records so could you remake it using just the GitHub UI?

Comment (staticsafe) [inline docs/Running-Mastodon/Production-guide.md]: @Gargron understood, I'll delete this fork, close this pull request and make a new one.

**Comment by staticsafe** [comment]
New pull request for this is #4132
